docker/postgres/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM postgres:9.6.3
```

Comment:
This file seems a bit redundant, could simply specify the image in the docker-compose.yml file

Reply:
I'm using this file now :)

Reply:
<3 @lboynton that fixed the problems I was having!